### PR TITLE
Support overriding resizer image

### PR DESCRIPTION
**Pull request by b-ranto**
Signed-off-by: Boris Ranto <branto@redhat.com>

### CODE REVIEW

File: hack/generate-latest-csv.sh [hunk 1]
```diff
@@ -44,6 +44,7 @@ $CSV_CHECKSUM \
 	--ceph-image="$CEPH_IMAGE" \
 	--rook-csi-ceph-image="$ROOK_CSI_CEPH_IMAGE" \
 	--rook-csi-registrar-image="$ROOK_CSI_REGISTRAR_IMAGE" \
+	--rook-csi-registrar-image="$ROOK_CSI_RESIZER_IMAGE" \
```
[the remark this answers is not in the document]
Comment (b-ranto) [reply hack/generate-latest-csv.sh]: Thanks for noticing, it should be fixed, now.

**Comment by b-ranto** [comment]
Repushed for the CI failure, it didn't update the csv-checksum source before.

File: tools/csv-checksum/csv-checksum.go [hunk 2]
```diff
@@ -114,6 +115,7 @@ func main() {
 	md5Hash = addArgToHash("--ceph-image", *cephContainerImage, md5Hash)
 	md5Hash = addArgToHash("--rook-csi-ceph-image", *rookCsiCephImage, md5Hash)
 	md5Hash = addArgToHash("--rook-csi-registrar-image", *rookCsiRegistrarImage, md5Hash)
+	md5Hash = addArgToHash("--rook-csi-resizer-image", *rookCsiRegistrarImage, md5Hash)
```
Comment (umangachapagain) [inline tools/csv-checksum/csv-checksum.go]:
```suggestion
	md5Hash = addArgToHash("--rook-csi-resizer-image", *rookCsiResizerImage, md5Hash)
```

Comment (b-ranto) [reply tools/csv-checksum/csv-checksum.go]: I have also noticed that, it should be fixed, now.

Comment (umangachapagain) [reply tools/csv-checksum/csv-checksum.go]: I still don't see the changes pushed.

Comment (b-ranto) [reply tools/csv-checksum/csv-checksum.go]: Ah, yes, it turned out I forgot to ammend the commit before pushing. :-/

**Pull request merged**